es/architecture.tex

```diff
@@ -162,26 +163,27 @@ \subsection*{Problemas de Parsons}
 
 \figimg{figures/frisian.png}{Construyendo una oración}{f:architecture-frisian}
 
-El equivalente de codificación de este ejemplo
+El equivalente en programación de este ejemplo
 se llama \gref{g:parsons-problem}{problema de Parsons}\footnote{Nombrado así debido a uno de sus creadores.}~\cite{Pars2006}.
 
 Cuando enseñes a programar,
 puedes darles a tus estudiantes las líneas de código que necesitan para resolver un problema
 y pedirles que las ordenen en el orden correcto.
 Esto les permite concentrarse en el flujo de control y en las dependencias de datos,
 sin distraerse con la denominación de las variables o tratando de recordar qué funciones llamar.
-Múltiples estudios han demostrado que los problemas de Parsons demandan menos tiempo de resolución pero producen resultados educativos equivalentes~\cite{Eric2017}.
+Múltiples estudios han demostrado que los problemas de Parsons demandan menos tiempo de resolución 
+y producen resultados educativos equivalentes~\cite{Eric2017}.
 
 
-\subsection*{Ejemplos desvanecidos}
+\subsection*{Ejemplos desvanecidos (o con espacios en blanco)}
 
 Otro tipo de ejercicio que se puede explicar en términos de carga cognitiva
-es dar a tus estudiantes una serie de ejemplos desvanecidos \grefdex{g:faded-example}{faded examples}{faded example}.
+es dar a tus estudiantes una serie de ejemplos desvanecidos \grefdex{g:faded-example}{\emph{faded examples en inglés}}{faded example}.
```

Comment:
puede ser que sobre un faded example? y "en inglés" lo dejaría sin itálicas.
Revisalo antes de aceptar porque me confunden las entradas a glosario en latex:

```suggestion
es dar a tus estudiantes una serie de ejemplos desvanecidos \grefdex{g:faded-example}(\emph{faded examples} en inglés).
```